### DISCUSSION TITLE
Number of mentored projects for proposal in GSoC 2021

### DISCUSSION THREAD

**Discussion by AlexVelezLl** [networkx/networkx#4676] [Q&A] [answered]
Hi! I'm a CS-student and I want to participate in GSoC 2021 and contribute to NetworkX. I went trought the Mentored Projects, but I have a question.  In my application, should I only solve one of the Mentored Projects, or I can solve more than one to enhance my proposal?

I am very grateful in advance :)

**Comment by Hetarth02** [comment]
I think you can solve more than one.

**Comment by dschult** [comment, marked as answer]
The guideline for student proposals is very flexible. (You can propose your own project.) 
But the time is finite (actually fairly limited) so make sure any proposal is reasonable given the time of the program.

**Reply by AlexVelezLl** [reply]
Ok, I'll make sure of that. Thanks for your time.

Regards.
